src/components/atoms/InputText/InputText.tsx

```diff
@@ -46,15 +56,14 @@ const Input = styled.input<InputProps & { theme: AppTheme }>`
   padding: 0 16px;
   padding-left: ${({ startIcon }) => !!startIcon && '60px'};
   padding-right: ${({ endIcon }) => !!endIcon && '60px'};
-  font-size: ${typography.sizes.text.body};
-  font-weight: ${typography.weights.regular};
+  font-size: ${({ theme, size = 'body' }) => theme.typography.text.sizes[size]};
```

Comment:
Add a default font-size of 16px

Reply:
'body' is equal to 16px so there is still a default of 16px this way